src/Templating/TemplateRegistry.php

```diff
@@ -77,17 +77,11 @@ public function __construct(array $templates = [])
         $this->templates = $templates;
     }
 
-    /**
-     * @return string[]
-     */
     public function getTemplates(): array
     {
         return $this->templates;
     }
 
-    /**
-     * @param string[] $templates
```

Comment:
why you remove this one?

Reply:
The same comment exists on the interface